region/maridia/inner-green/Pants Room.json

```diff
@@ -437,7 +443,8 @@
               "ScrewAttack",
               "canStaggeredWalljump"
             ]}
-          ]}
+          ]},
+          "h_pauseAbuseMinimalReserveRefill"
```

Comment:
I tried all of the changes in this PR except this one. I wasn't sure how to grapple jump.